index.js

```diff
@@ -267,11 +240,21 @@ class ClientServiceBase {
    * @throws {ClientError|Error}
    */
   async _createContractsListingRequest(contractId) {
+    const validator = new ClientPropertiesValidator([
+      'scalar.dl.client.cert_holder_id',
```

Comment:
Can we make these strings constant since they are repeatedly used ? (like CERT_HOLDER_ID)